src/hooks/useDarkMode.ts

```diff
@@ -1,16 +1,27 @@
 import React, { useEffect, useState } from 'react'
-// React.Dispatch<React.SetStateAction<string>>
+import colourThemes from 'assets/themes.json'
 export default function useDarkMode(): [string, (theme: string) => void] {
   const [theme, setTheme] = useState(localStorage.getItem('carrotTheme') || 'light');
-  const colorTheme = (theme === 'light') ? 'dark' : 'light'
   const themeSet = (theme: string) => {
     setTheme(theme)
     localStorage.setItem('carrotTheme', theme)
   }
   useEffect(() => {
+    const applyTheme = localStorage.getItem('carrotTheme') || 'light'
+    setTheme(applyTheme)
+    console.log('setting the theme', applyTheme)
+    const colourThemesJSON: any = colourThemes;
+    const colours = colourThemesJSON[applyTheme]

```

Comment:
make sure you have a color theme to fallback on in case applyTheme is invalid (very possible if user messes with localstorage)

Reply:
mannnnnnnn